Core1Platform/AdministratingCore1/03_ManageGroups.md

```diff
@@ -0,0 +1,186 @@
+    A further drop-down list is displayed. Repeat steps **4** and **5** to add as many users as necessary.
+
+6. Click the [SAVE] button in the upper right corner of the *Add user* window.  
+    The users have been saved to the group and are displayed in the list of users.
+
+7. Click the [SAVE] button in the upper right corner of the *Edit group* window.   
+    The *Group saved* pop-up window is displayed. The changes to the group have been saved.
+
+    ![Group saved](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupSaved.png "[Group saved]")
+
+
+## Remove user from a group via the group management
+
+A user is active in an account as long as the user is assigned a user group.    
+If you want to remove a user from a sandbox, you must remove the user from all groups to which he &frasl; she is assigned. There are the following limitations:
+- You cannot remove a user from the ActindoAdmingroup in a sandbox of which he or she is the owner. <!--- Kann man den owner ändern? -->
+- You cannot remove a user from all groups in an account in which the user has been created. In this case, you have to inactivate him&frasl;her, see [Edit user](./02_ManageUsers.md#edit-user).   
+
+Note that it is not possible to remove a group assignment by delimiting the validity end date to a specific date. If you want to change the validity period, you must remove the group assignment and create a new one with a limited validity end date.
+
+
+#### Prerequisites
+
+- You are granted to remove the user assignment from a group.
+
+#### Procedure
+
+*Settings > Users and groups > Tab GRUPPENVERWALTUNG (GROUP MANAGEMENT)*
+
+![Group management](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupManagment.png "[Group management]")
+
+1. Select the group, from which you want to remove a user.   
+    The editing toolbar is displayed.
+
+   ![Group selected](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupSelected.png "[Group selected]")
+
+2. Click the ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit) button.
+    The *Edit group "Group name"* view is opened. By default, the *Users* tab is displayed. 
+
+   ![Group selected](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupManagementUsers.png "[Group selected]")
+
+3. Select a user.   
+    The editing toolbar is displayed.
+
+   ![User selected](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupUserSelected.png "[User selected]")
+
+4. Click the ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete) button.   
+    The selected user is removed from the list.
+
+5. Click the [SAVE] button.   
+    The *GRUPPENVERWALTUNG (GROUP MANAGEMENT)* view is displayed again.
+
+
+## Remove user group
+
+If a user group is no longer needed, you can remove it. Note, groups are removed regardless of whether users are still assigned.
```

Comment:
```suggestion
If a user group is no longer needed, you can delete it. Note that groups are removed regardless of whether users are still assigned.
```
Werden die Benutzer, die in der Gruppe sind, auch gelöscht? Ist es wirklich nötig zu prüfen, ob Benutzer zugewiesen sind oder nicht, wie in Punkt 1 beschrieben?

Reply:
Wäre halt blöd, wenn man eine Gruppe löscht, uzu der noch User zugewiesen sind, ohne eine neue zu generieren

Reply:
done